### DISCUSSION TITLE
Make it possible to add/remove links to an existing lab, without bringing everything down/up

### DISCUSSION THREAD

**Discussion by jbemmel** [srl-labs/containerlab#718] [Ideas]
Based on a comment at https://stubarea51.net/2021/11/23/network-modeling-segmented-lab-access-with-containerlab-and-zerotier I was thinking it would be feasible to add a command to insert/remove links on the fly, for those people who just spent 30 minutes building a 500-node lab and notice they forgot a link or two

**Comment by hellt** [comment]
that would be a nice addition, but it requires some clever diff'ing
when you change a topo and want the running lab to reconcile to the new state you need to know exactly what has changed
